ral-facilities/operationsgateway
# DSEGOG-194 hiding gridlines for second y-axis

## Description
Fixes bug where these weren't hidden along with the rest.

## Testing instructions
Check gridlines are hidden locally and also make sure the grid option changes correctly for the y2 axis in the amended snapshot.


## Agile board tracking
Closes DSEGOG-194
